web/src/components/profile/ProfileAddressFields.test.tsx

```diff
@@ -0,0 +1,132 @@
+import { ProfileAddressFields } from "@/components/profile/ProfileAddressFields";
+import { generateAddress } from "@/test/factories";
+import { WithStatefulAddressData } from "@/test/mock/withStatefulAddressData";
+import { Address, emptyAddressData, Municipality } from "@businessnjgovnavigator/shared/";
+import { generateMunicipality } from "@businessnjgovnavigator/shared/test";
+import { fireEvent, render, screen, within } from "@testing-library/react";
+
+describe("<ProfileAddressField  />", () => {
```

Comment:
Note, per previous comment move validation tests to this test file.

Reply:
tests were moved to the profile-owning so we could validate the rendering the alerts

Reply:
added tests in the Profile NJ component to test error labels/ helper text